ReactiveCocoaTests/Swift/SignalProducerLiftingSpec.swift

```diff
@@ -298,6 +298,48 @@ class SignalProducerLiftingSpec: QuickSpec {
 				expect(lastValue).to(equal(1))
 			}
 		}
+		
+		describe("skipUntil") {
+			var producer: SignalProducer<Int, NoError>!
+			var observer: Signal<Int, NoError>.Observer!
+			var triggerObserver: Signal<(), NoError>.Observer!
+			
+			var lastValue: Int? = nil
+			
+			beforeEach {
+				let (baseProducer, baseIncomingObserver) = SignalProducer<Int, NoError>.buffer()
+				let (triggerSignal, incomingTriggerObserver) = SignalProducer<(), NoError>.buffer()
+				
+				producer = baseProducer.skipUntil(triggerSignal)
+				observer = baseIncomingObserver
+				triggerObserver = incomingTriggerObserver
+				
+				lastValue = nil
+				
+				producer.start { event in
+					switch event {
+					case let .Next(value):
+						lastValue = value
+					default:
+						break
+					}
+				}
+			}
+			
+			it("should skip values until the trigger fires") {
```

Comment:
Can you write another test for when the `trigger` completes without sending any events?